articles/active-directory-b2c/tutorial-single-page-app.md

```diff
@@ -99,9 +98,10 @@ Now that you've obtained the sample, update the code with your Azure AD B2C tena
             storeAuthStateInCookie: true
         }
     };
+
     ```
 
-    The name of the user flow used in this tutorial is **B2C_1_signupsignin1**. If you're using a different user flow name, specify that name in the `authority` value.
+    The name of the user flow used in this tutorial is **b2c_1_susi**. If you're using a different user flow name, specify that name in the `authority` value.
```

Comment:
We need to stick with **B2C_1_signupsignin1** for the SUSI policy name. In the [user flow tutorial](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-user-flows) and several other articles, this is the name of the SUSI flow that the user creates, and the name that all other tutorials reference.